### ISSUE TITLE
instance path is not set for the flask app

### ISSUE THREAD

**Issue by jma** [inveniosoftware/pytest-invenio#17]
The `instance_path` is only used for the DB not for the complete app. The consequence is that `invenio.cfg` is used for the tests. My actual workaround `conftests.py` is the following:
```python
@pytest.fixture(scope='module')
def create_app(instance_path):
    """Create test app."""
    from invenio_app.factory import create_api
    return create_api
```

**Comment by lnielsen** [comment]
I'm trying to understand where the problem lies. Do you have a tiny example that fails?

Could look like an issue between invenio-app and pytest-invenio. Basically ``instance_path`` fixture get's called everytime you need an app (app_config -> db_uri -> instance_path), but I assume then problem is that the instance path is not injected into the Flask app when it's created (or there's some sort of clean-up issue).

**Comment by lnielsen** [comment]
Closed via inveniosoftware/invenio-base#142 and inveniosoftware/invenio-app#47
